knowledge-base/how-to-migrate-minio-storage.md

```diff
@@ -0,0 +1,106 @@
+# How to migrate MinIO storage
+https://docs.kubermatic.com/kubermatic/v2.23/installation/upgrading/upgrade-from-2.22-to-2.23/#minio-upgrade
+
+## boot new deployment
+
+Get the deployment, svc, and pvc yaml files
+```bash
+kubectl -n minio get deployments.apps minio -o yaml > deployment.yaml
+kubectl -n minio get svc minio -o yaml > svc.yaml
+kubectl -n minio get pvc minio-data -o yaml > pvc.yaml
+```
+Change the name of the deployment, pod spec and pvc to minio-new
+But don't change the namespace!
+
+apply them:
+```bash
+kubectl -n minio apply -f deployment.yaml
+kubectl -n minio apply -f svc.yaml
+kubectl -n minio apply -f pvc.yaml
+```
+
+## Start debug container on cluster
```

Comment:
you could use directly the minio image as well, I using locally ofte
```
docker run --rm -it --network host -v "`PWD`":/data -e MC_HOST_minio=https://___ACCESS__KEX___:___SECRET_ACCESS_KEY____@minio.run-2.lab.kubermatic.io --name run-2-mc --entrypoint bash minio/mc
```

Reply:
hm, i tried it but struggled to get into a shell, which is actually more convenient to do all the steps. Would be much better than needing to install something